src/AMPGO02.jl

```diff
@@ -0,0 +1,23 @@
+# A one dimensional optimization problem
+#
+# Problem 2 in
+# http://infinity77.net/global_optimization/test_functions_1d.html#d-test-functions
+# Andrea Gavana
+#
+# S. Goyette, Sherbrooke 2016/2017
+
+export AMPGO02
+
+function AMPGO02(n::Int64=1)
+   nlp = Model()
```

Comment:
`n` should not be a parameter that the user can change if the problem size is fixed.